demo/px-data-grid-remote-data-provider-demo.html

```diff
@@ -157,6 +157,10 @@
             callback(JSON.parse(xhr.responseText), size);
           };
 
+          // In case of network issues, pass the empty array
+          // When network will be good, px-data-grid will try to get the same chunk of data again
```

Comment:
`When network will be good` => `When network is good`

Reply:
done.